manifests/service.pp

```diff
@@ -1,19 +1,18 @@
 # Resource for managing the FluentD service
-class fluentd::service inherits fluentd {
+class fluentd::service {
+  include fluentd
   if $fluentd::service_manage {
-    # On Windows
-    # Fluentd v3 and older required us to self-register the Windows services
-    # In Fluentd v4+, the Windows service is registered as part of the MSI installer
-    if ($facts['os']['family'] == 'windows' and versioncmp($fluentd::repo_version, '3') <= 0) {
```

Comment:
If we are going to remove this code we need to put a note in the readme that we are officially dropping support for Fluentd 3